plugins/field-slider/test/field_test_helpers.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const assert = require('chai').assert;
+const Blockly = require('blockly');
```

Comment:
Did we figure out whether we need this?

Reply:
I actually added this to make my IDE happy for lint reasons, but it works without it (and also there's other lint things my IDE is complaining about). So, I'm going to remove.